packages/strapi-admin/admin/src/themes/colors.js

```diff
@@ -10,6 +10,8 @@ const colors = {
   pink: '#ff5b77',
   purple: '#613d7c',
   gray: '#464a4c',
+  border: '#E3E9F3',
+  'border-active': '#78caff',
```

Comment:
It's probably more practical to use camelCase for the theme, so we can call it easily (like `colors.borderActive` instead of `colors['border-active']`). 
We still have some old names like below but I think we won't keep it with the new design system.